### DISCUSSION TITLE
How to drop unwanted/excess information from alert that is not a part of alert_text_args ?

### DISCUSSION THREAD

**Discussion by roman-tasi** [jertel/elastalert2#1174] [Q&A] [answered]
From this image I have a Discord alert that has excess information that I don't want in the alert. Basically everything under `"Reason - The specified account's password has expired."` is unwanted:

![viber_image_2023-04-28_12-58-32-738](https://user-images.githubusercontent.com/91568507/235267199-85fe2627-6c19-425f-9f0a-ac58001524ae.jpg)

This is my `alert_text` field in the corresponding rule file:
```
alert_text: |-
   This log is generated by - {0}
   Reason - {1}
```
So is there a way to eliminate all the info that is not a part of the `alert_text` ?

Also I want to mention I only notice this excess information for the Slack & Discord alerts, not the Email alerts.

**Comment by jertel** [comment, marked as answer]
Add `alert_text_type: alert_text_only` to your rule.
